cmake/XercesIntTypes.cmake

```diff
@@ -58,14 +58,14 @@ set(HAVE_OFF_T ${SIZEOF_OFF_T})
 set(HAVE_SIZE_T ${SIZEOF_SIZE_T})
 set(HAVE_SSIZE_T ${SSIZEOF_SSIZE_T})
 set(HAVE_WCHAR_T ${WCHAROF_WCHAR_T})
-if(SIZEOF_SIZE_T)
+if(HAVE_SIZEOF_SIZE_T)
```

Comment:
I'm not sure where these changes have come from.  Is this a reversion of an earlier change?

If the history is messy, it might be easiest to rebase all your changes onto the current `master` and start a new PR.